profile/README.md

```diff
@@ -30,6 +30,8 @@ Every year, researchers at Utrecht University publish thousands of scientific re
 
```

Comment:
I think a small title to explain the cards would be nice
```suggestion
#### Featured projects
```

Reply:
Great feedback. Based on your comment, I changed the update cycle to weekly instead of biweekly. Now we can name this section something like "Feature projects this week".